e2etest/zt_newe2e_blob_test.go

```diff
@@ -12,136 +10,200 @@ func init() {
-	fileName := "test_page_blob.vHd"
-	body := NewRandomObjectContentContainer(svm, common.KiloByte)
-	tier := ResolveVariation(svm, []common.PageBlobTier{common.EPageBlobTier.P10(), common.EPageBlobTier.P20(), common.EPageBlobTier.P30(), common.EPageBlobTier.P4(), common.EPageBlobTier.P40(), common.EPageBlobTier.P50()})
-
-	srcObj := CreateResource[ContainerResourceManager](svm, GetRootResource(svm, common.ELocation.Local()), ResourceDefinitionContainer{}).
-		GetObject(svm, fileName, common.EEntityType.File())
-
-	srcObj.Create(svm, body, ObjectProperties{})
-
-	dstContainer := CreateResource[ContainerResourceManager](svm, GetRootResource(svm, common.ELocation.Blob(), GetResourceOptions{PreferredAccount: pointerTo(PremiumPageBlobAcct)}), ResourceDefinitionContainer{})
-	dstObj := dstContainer.GetObject(svm, fileName, common.EEntityType.File())
-
-	RunAzCopy(svm, AzCopyCommand{
-		Verb:    AzCopyVerbCopy,
-		Targets: []ResourceManager{srcObj, dstObj.(RemoteResourceManager).WithSpecificAuthType(EExplicitCredentialType.SASToken(), svm, CreateAzCopyTargetOptions{})},
-		Flags: CopyFlags{
-			CopySyncCommonFlags: CopySyncCommonFlags{
-				Recursive: pointerTo(true),
-			},
-			BlobType:     pointerTo(common.EBlobType.PageBlob()),
-			PageBlobTier: pointerTo(tier),
-		},
-	})
-
-	ValidateResource[ObjectResourceManager](svm, dstObj, ResourceDefinitionObject{
-		Body: body,
-		ObjectProperties: ObjectProperties{
-			BlobProperties: BlobProperties{
-				Type:                pointerTo(blob.BlobTypePageBlob),
-				BlockBlobAccessTier: pointerTo(tier.ToAccessTierType()),
-			},
-		},
-	}, true)
-}
-
-func (s *BlobTestSuite) Scenario_UploadBlob(svm *ScenarioVariationManager) {
-	// Scale up from service to object
-	dstObj := CreateResource[ContainerResourceManager](svm, GetRootResource(svm, common.ELocation.Local()), ResourceDefinitionContainer{}).GetObject(svm, "test", common.EEntityType.File())
-	body := NewRandomObjectContentContainer(svm, SizeFromString("1K"))
-	// Scale up from service to object
-	srcObj := CreateResource[ObjectResourceManager](svm, GetRootResource(svm, common.ELocation.Blob()), ResourceDefinitionObject{
-		ObjectName: pointerTo("test"),
-		Body:       body,
-	})
-
-	RunAzCopy(
-		svm,
-		AzCopyCommand{
-			Verb: AzCopyVerbCopy,
-			Targets: []ResourceManager{
-				srcObj,
-				TryApplySpecificAuthType(dstObj, ResolveVariation(svm, []ExplicitCredentialTypes{EExplicitCredentialType.OAuth(), EExplicitCredentialType.SASToken()}), svm, CreateAzCopyTargetOptions{}),
-			},
-			Flags: CopyFlags{
-				CopySyncCommonFlags: CopySyncCommonFlags{
-					Recursive: pointerTo(true),
-				},
-			},
-		})
-
-	ValidateResource[ObjectResourceManager](svm, dstObj, ResourceDefinitionObject{
-		Body: body,
-	}, true)
-}
-
-func (s *BlobTestSuite) Scenario_DownloadBlob(svm *ScenarioVariationManager) {
+// func (s *BlobTestSuite) Scenario_UploadBlockBlobs(svm *ScenarioVariationManager) {
```

Comment:
Why are all these test cases commented here?

Reply:
commented as of now. To validate the new test case added.
Will remove these changes and create a new ticket for adding a TC for this scenario.